build/template-OneBranch-Tests-libsandsamples.yaml

```diff
@@ -14,9 +14,10 @@ steps:
   inputs:
     version: 6.x
 
-- template: template-restore-build-libsandsamples.yaml
+- template: template-OneBranch-CI-libsandsamples.yaml

```

Comment:
`template-restore-build-libsandsamples.yaml` is used by our [PR builds](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/dc93a5469a886dfc4a8dd17d00fae540503884da/build/template-build-and-prep-automation.yaml#L14), which points to nuget.org (it can). But we do not want OB to point to nuget.org. When we separated OB build and run as two tasks we ended up using the PR yaml for tests, instead we need to point to `template-OneBranch-CI-libsandsamples.yaml` which uses IDDP nuget feed.

Reply:
but the template `template-OneBranch-CI-libsandsamples.yaml` itself is not prepped for test runs it was very build specific, hence made some changes to the yaml so we can re-use the same for OB build and tests